File: 11INFORMATIKA/ANDROID/11KOM2/01/Employee.java

public class Employee {

    public static int numberOfEmp = 0;
    private String name;

    // CONSTRUCTOR
    public Employee(){
        numberOfEmp++;
    }

    public Employee(String name){
        this.name = name;
        numberOfEmp++;
    }

    // ACCESSOR
    public String getname(){
        return this.name;
    }

    // MUTATORS
    public void setName(String new_name){
        this.name = new_name;
    }

    // STATIC METHOD
    public static int getNumberOfEmp(){
        return numberOfEmp;
    }

    public static void main(String[] args) {
        Employee emp1 = new Employee();
        Employee emp2 = new Employee("Jane");

        System.out.println(Employee.getNumberOfEmp());

        emp1.setName("John");
        System.out.println(emp1.getname());
    }

    
}

File: 11INFORMATIKA/ANDROID/11KOM2/01/employee.py

class Employee():

    number_of_emp = 0

    # CONSTRUCTOR
    def __init__(self, name=""):
        self.name = name
        Employee.number_of_emp += 1 # static attr

    # GETTER / ACCESSOR
    def get_name(self):
        return self.name
    
    # SETTER / MUTATOR
    def set_name(self, new_name):
        self.name = new_name

    @staticmethod
    def get_number_of_emp():
        return Employee.number_of_emp
    
if __name__ == "__main__":

    emp_1 = Employee()
    emp_2 = Employee("Jane")

    print(Employee.get_number_of_emp())

    emp_1.set_name("John")
    print(emp_1.get_name())
    
